spec/features/dashboard_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+feature "dashboard" do
+  
+  before(:each) do
+    visit "/"
+  end
+  context "when not logged in" do
+    it "redirects to index" do
+      visit "/dashboard"
+      
+      expect(current_path).to eq('/')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.